l10n_fr_ape/models/res_partner.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2020 MonsieurB <monsieurb@saaslys.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResPartnerIndustry(models.Model):
+    _inherit = "res.partner.industry"
+
+    code = fields.Char(string='Code')
```

Comment:
I think the name should contain "ape".

Reply:
The model already exist but the code is in the full_name field . like C MANUFACTURING
so i had search by code features with more of 7 undred...

Reply:
other country may have other code name not "activité Principal de Etablissement" like unece
https://github.com/OCA/community-data-files/tree/12.0/base_unece

Reply:
Well. It's not consistent with the name of the module, that contains ape.

Reply:
Doesn't it make sense to add this field in a separate module in partner-contact repository?
ie: `partner_industry_code`

I'm thinking this could be reused by other localization modules

btw, no need to explicitly set the `string` if it's the same than the field name
```suggestion
    code = fields.Char()
```